utils/smartpaste/githubcode.tsx

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Rule} from 'turndown';
+
+export function githubCodeTurndownRuleBuilder(hasFirstPart: boolean, hasLastPart: boolean, text: string): Rule {
+    return {
+        filter: (node: Node): boolean => {
+            return (node.parentNode as HTMLElement)?.id === 'turndown-root' && node.nodeName === 'TABLE' && (/\b(js|blob|diff)-./).test((node as HTMLTableElement).className);
```

Comment:
Why are we checking the classname of the node before determining if we should run this rule?

Reply:
This rule is a migration from the GitHub code paste support that existed before, that check is the same one that was already there but now though turndown library.